docs/custom-images-contract.rst

```diff
@@ -0,0 +1,113 @@
+.. _custom-images-contract:
+
+======================
+Custom Images contract
+======================
+
+The contract is a set of requirements for crawler custom docker images to comply running with Scrapy Cloud.
```

Comment:
"A contract is a set of requirements that any crawler custom Docker image have to comply with to be able to run on Scrapy Cloud. If you want to run crawlers built using other framework/language than Scrapy/Python, you have to make sure your image follows the contract statements listed below."

Thoughts?